docs/install/_initial_setup.rst

```diff
@@ -21,55 +21,58 @@ To do the initial setup of your Kolibri facility after the installation, follow
 	  The default language configured for content in your browser preferences might override the language you choose in this step. To ensure that Kolibri displays in the desired language, make sure to configure it as default in the browsers of all the devices that will be used to view Kolibri content.
 
 
-#. **Name your Facility.** A "Facility" is the location where you are installing Kolibri, such as a school or a training center.
+#. **Facility name and type.** A "Facility" is the location where you are installing Kolibri, such as a school, training center, or even your home. 
```

Comment:
One subtle change in 0.11 is that for the 'personal' facility type, we no longer ask for a facility name. (it's auto-generated under the hood)

Reply:
Added, thnx!